src/components/ec-submenu/ec-submenu.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div
+    ref="submenu"
+    v-if="submenu"
+    class="ec-submenu"
+  >
+    <div
+      ref="header"
+      class="ec-submenu__header-container"
+    >
+      <ul
+        class="ec-submenu__header"
+        size="24"
+      >
+        <li
+          v-for="(menuItem, index) in submenu"
+          :key="menuItem.index"
+          class="ec-submenu__header-item"
+          :class="{'ec-submenu__header-item--active': index === activeIndex}"
+          @click="$emit('change', index)"
+        >
+          <a
+            v-if="menuItem.href"
+            :href="menuItem.href"
+            class="ec-submenu__header-title"
+            @click.prevent
```

Comment:
If I remove this instead to change tab it opens a new page.

Reply:
In that case we should not allow to pass any href to the menuItem and make it always `#`. Maybe the logic should be: If there is a route, render router-link (no prevent is needed there), otherwise render anchor with `#` and prevent.

Reply:
i removed totally the href from a. It still loads another page. If you remove it doesn't